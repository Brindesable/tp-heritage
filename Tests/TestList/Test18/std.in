C rond 0 0 10
C cercle 0 0 5
C disque 0 0 3
LIST