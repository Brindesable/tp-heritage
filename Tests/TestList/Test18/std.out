OK
#1 object(s) created
OK
#1 object(s) created
OK
#1 object(s) created
C cercle 0 0 5
C disque 0 0 3
C rond 0 0 10
